### PR TITLE
Updated redirects file for URLs under "about"

### LINKED ISSUE #237: Update stale redirects
## Description

A few of the redirects in our _redirects file are now stale, since the "about" pages now appear under an "about" directory structure.

## Solution

Update the redirects to reflect the changed structure for pages under "about."

### PR DESCRIPTION
**Pull request by smythp**
## Summary

Some previous first-level URLs are now under "about". Added redirects for these so that inbound links will still work.

## Notes

Resolves #237

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #redirect file for moved web pages.
-/main-concepts                              /tooling
-/history                                    /research
-/cosign/overview                            /signing/quickstart    
+/main-concepts                              /about/tooling
+/security                              /about/security
+/threat-model                              /about/threat-model/
```
Comment (ltagliaferri) [inline static/_redirects:4]:
```suggestion
/threat-model                               /about/threat-model/
```

File: static/_redirects [hunk 2]
```diff
@@ -1,7 +1,9 @@
 #redirect file for moved web pages.
-/main-concepts                              /tooling
-/history                                    /research
-/cosign/overview                            /signing/quickstart    
+/main-concepts                              /about/tooling
+/security                              /about/security
```
Comment (ltagliaferri) [inline static/_redirects:3]:
```suggestion
/security                                  /about/security
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Just fixing the alignment here

File: static/_redirects (hunk 1 above)
Comment (smythp) [reply static/_redirects:4]: Looks good. Going to have to be multiple commits as GH isn't letting me batch (probably related to the signoff).

File: static/_redirects [hunk 3]
```diff
@@ -1,7 +1,9 @@
 #redirect file for moved web pages.
-/main-concepts                              /tooling
-/history                                    /research
-/cosign/overview                            /signing/quickstart    
+/main-concepts                              /about/tooling
+/security                                  /about/security
```
Comment (ltagliaferri) [inline static/_redirects:3]: This one still is one space out 🤔 

```suggestion
/security                                   /about/security
```

**Review by ltagliaferri — APPROVED** [review]

File: static/_redirects (hunk 3 above)
Comment (smythp) [reply static/_redirects:3]: Maybe I should have seen if there was an Emacs mode for weird unspecified URL lists, ha.

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**